### PR TITLE
test: check type coverage

### PR DESCRIPTION
**Pull request by ChristianMurphy**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -34,6 +34,7 @@ export function zwitch(key, options) {
    * and it’s result is returned.
    *
    * @param {...unknown} [value]
+   * @this {Options}
```
Comment (wooorm) [inline index.js]:
```suggestion
   * @this {unknown}
```

The this value could be anything, as is passed through here. Similar to how all args are passed through!

File: package.json [hunk 2]
```diff
@@ -34,6 +34,7 @@
     "remark-preset-wooorm": "^8.0.0",
     "rimraf": "^3.0.0",
     "tape": "^5.0.0",
+    "type-coverage": "^2.16.3",
```
Comment (wooorm) [inline package.json]:
```suggestion
    "type-coverage": "^2.0.0",
```

File: package.json [hunk 3]
```diff
@@ -44,7 +45,8 @@
     "format": "remark . -qfo && prettier . -w --loglevel warn && xo --fix",
     "test-api": "node test.js",
     "test-coverage": "c8 --check-coverage --branches 100 --functions 100 --lines 100 --statements 100 --reporter lcov node test.js",
```
Comment (wooorm) [inline package.json]:
```suggestion
    "test-coverage": "c8 --check-coverage --branches 100 --functions 100 --lines 100 --statements 100 --reporter lcov node test.js && type-coverage",
```

Here maybe? Or in `format` or `build`?

File: index.js (hunk 1 above)
Comment (ChristianMurphy) [reply index.js]: updated

File: package.json (hunk 3 above)
Comment (ChristianMurphy) [reply package.json]: updated

> format

maybe? it doesn't modify/fix content in the way the other formatters do

> build

Sure, added a post build step

File: package.json (hunk 2 above)
Comment (ChristianMurphy) [reply package.json]: updated

**Review by wooorm — APPROVED** [review]

**Pull request merged**